data/src/main/kotlin/com/vultisig/wallet/data/repositories/swap/SwapKitProviderCache.kt

```
@@ -67,42 +92,72 @@ internal class SwapKitProviderCacheImpl @Inject constructor(private val api: Swa
             }
             try {
                 val response = api.providers()
                 val chains = response.toEnabledChains()
                 enabledChains = chains
-                fetchedAtMillis = nowInner
+                retryAfterMillis = 0
+                fetchedAtMillis = now
                 chains
             } catch (e: CancellationException) {
                 throw e
             } catch (e: SwapKitError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)